### PR TITLE
Register new package Bridge.jl v0.4.1

### PR DESCRIPTION
**Pull request by attobot**
Repository: [mschauer/Bridge.jl](https://github.com/mschauer/Bridge.jl)
Release: [v0.4.1](https://github.com/mschauer/Bridge.jl/releases/tag/v0.4.1)
cc: @mschauer

Please make sure that:
- CI passes for supported Julia versions (if applicable).
- Version bounds reflect minimum requirements.

@mschauer This PR will remain open for 24 hours for feedback (which is optional). If you get feedback, please let us know if you are making changes, and we'll merge once you're done.

### CODE REVIEW

**Comment by mschauer** [comment]
Replaces #10005. Thank you to @tkelman for helpful comments.

**Comment by mschauer** [comment]
Julia 0.7 build failure is due to JuliaStats/Distributions.jl#644

**Comment by mschauer** [comment]
I am making some changes

File: Bridge/versions/0.4.1/requires [hunk 1]
```diff
@@ -0,0 +1,5 @@
+julia 0.6
+Distributions
+StaticArrays
+Polynomials
+Compat
```
Comment (tkelman) [inline Bridge/versions/0.4.1/requires]: It's best to declare a minimum version dependency that implements the features you need.

Comment (mschauer) [inline Bridge/versions/0.4.1/requires]: Yes, but this is possibly the next release of Compat, the next release of StaticArrays and the next release of Distributions ;-)

Comment (tkelman) [inline Bridge/versions/0.4.1/requires]: right, it looks like the package isn't passing tests until you get those packages to tag?

Comment (mschauer) [inline Bridge/versions/0.4.1/requires]: yes, it passed test locally on my machine with all the PR branches checked out

Comment (mschauer) [inline Bridge/versions/0.4.1/requires]: [*] Distributions v0.14.2

**Comment by mschauer** [comment]
Waiting for Compat.jl getting tagged.

**Comment by mschauer** [comment]
> All checks have passed.

**Pull request merged**